### PR TITLE
Remove VVV time zone format

### PR DESCRIPTION
**Pull request by sffc**
Unblocks #987
See #5658

### CODE REVIEW

File: components/datetime/src/time_zone.rs
Comment (robertbastian) [inline components/datetime/src/time_zone.rs:47]: nit: remove this, it's implemented

actually, do we need the full exemplar city names for it?

File: components/datetime/src/time_zone.rs
Comment (robertbastian) [inline components/datetime/src/time_zone.rs:170]: issue: remove this and `ExemplarCityFormat`, because that is what's actually blocking

File: components/datetime/src/time_zone.rs
Comment (sffc) [reply components/datetime/src/time_zone.rs:47]: hmm, that's annoying. Maybe not though because it seems Generic Partial Location is only used when you're not the primary zone?

**Comment by sffc** [comment]
I'll let you merge this @robertbastian if you conclude that it is what we need for #5657

**Review by robertbastian — APPROVED** [review]

File: components/datetime/src/time_zone.rs
Comment (robertbastian) [reply components/datetime/src/time_zone.rs]: I think it's fine, because #5657 uses the new logic for the bracketed part, i.e. it will say `Central European Time (Germany)` and `Pacific Time (Los Angeles)`.

Comment (robertbastian) [reply components/datetime/src/time_zone.rs]: It would actually be nicer to say `Pacific Time (United States)`, but I'd have to check if the metazone invariants allow that.

Comment (sffc) [reply components/datetime/src/time_zone.rs]: `Pacific Time (United States)` sounds extremely weird.

Comment (robertbastian) [reply components/datetime/src/time_zone.rs]: Ah, `Mountain Time (United States)` doesn't work.

Comment (robertbastian) [reply components/datetime/src/time_zone.rs]: > Pacific Time (United States) sounds extremely weird.

Weirder than `Pacific Time (Los Angeles)`? It's the Pacific as used in the United States, following United States DST rules. They're not Los Angeles DST rules. Does `Pacific Time (Mexico)` sound weird to you as well?

Comment (sffc) [reply components/datetime/src/time_zone.rs]: I think we're safe there because we only do the region override if the time zone is the only one in the country, so the city and region are interchangeable. Right?

Comment (robertbastian) [reply components/datetime/src/time_zone.rs]: > I think we're safe there because we only do the region override if the time zone is the only one in the country, so the city and region are interchangeable. Right?

Yes, current behaviour is safe. The proposed behaviour of `Pacific Time (Canada)` needs more work, because we need to know whether a metazone+country combination is unique. `ampa`+`US` is, `ammo`+`US` isn't.

**Pull request merged**
